src/task_info.rs

```diff
@@ -165,12 +165,15 @@ mod test {
     }
 
     #[meilisearch_test]
-    async fn test_failing_task(client: Client, movies: Index) -> Result<(), Error> {
-        let task_info = movies.set_ranking_rules(["wrong_ranking_rule"]).await?;
+    async fn test_failing_task(client: Client, index: Index) -> Result<(), Error> {
+        let task_info = client
+            .create_index("meilisearch_sdk-task_info-test-test_failing_task", None)
+            .await
+            .unwrap();
```

Comment:
Where did you create the index in the first place if it received `IndexAlreadyExists`?